cli/lib/commands/download.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = download;
 
-const { createReadStream, promises: fs } = require('fs');
+const { createReadStream, promises: fs } = require('fs-extra');
```

Comment:
TIL about fs-extra! Does this wrap the `{promises}` api in `graceful-fs` as well?

Reply:
It's just a [getter to the original `fs.promises`](https://github.com/jprichardson/node-fs-extra/blob/9f1c02916bb51d43feb7063f00f138e232f4d518/lib/index.js#L23-L28) unfortunately. I mostly added `fs-extra` for the `ensureDir` method, but was planning to change `promises` usage to just `fs-extra` methods after more tests are in place 😄